### PR TITLE
infra: fix Travis problem with JAVA_HOME

### PR DESCRIPTION
**Pull request by romani**
after doing : https://github.com/checkstyle/checkstyle/pull/13510
and loooking at logs
https://app.travis-ci.com/github/checkstyle/checkstyle/builds/265020910#L330

so finally jdk installer fixed problem with javahome.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -36,27 +36,19 @@ jobs:
...
 script:
-  # manually set JAVA_HOME to overcome issue with travis ci noted at
-  # https://github.com/checkstyle/checkstyle/pull/11699#issue-1261272652
-  # - JAVA_HOME='/usr/lib/jvm/adoptopenjdk-11-hotspot-ppc64el'
-  # - ./.ci/travis.sh init-m2-repo
-  # - ./.ci/travis.sh run-command "$CMD"
+  - ./.ci/travis.sh init-m2-repo
+  - ./.ci/travis.sh run-command "$CMD"
   # - ./.ci/travis.sh remove-custom-mvn
   # - ./.ci/travis.sh remove-adoptium-jdk
```
[the remark this answers is not in the document]
Comment (romani) [reply .travis.yml:45]: We do not use any special jdk and maven versions, install-xxxxx are not required also.

[the remark this answers is not in the document]
Comment (romani) [reply .travis.yml:45]: Remove install of: xsltproc ant

[the remark this answers is not in the document]
Comment (romani) [reply .travis.yml]: done.

**Review by nrmancuso — APPROVED** [review]

**Comment by nrmancuso** [comment]
Failure in semaphore is not related to changes here, looks like we are cancelling jobs based on some concurrency limit.

**Pull request merged**